### PR TITLE
CSPL-2167 - Make strict flag as default flag for essinstall

### PR DESCRIPTION
**Pull request by kumarajeet**
CSPL-2167 - make essinstall default ssl_enablement flag as strict to adhere with splunk doc
https://docs.splunk.com/Documentation/ES/7.0.2/Install/InstallEnterpriseSecuritySHC#Installing_Splunk_Enterprise_Security_from_the_command_line_in_a_search_head_cluster_environment

### CODE REVIEW

**Review by gaurav-splunk — APPROVED** [review]

File: api/v4/common_types.go [hunk 1]
```diff
@@ -387,12 +387,12 @@ type EsDefaults struct {
 	//     strict: Ensure that SSL is enabled
 	//             in the web.conf configuration file to use
 	//             this mode. Otherwise, the installer exists
-	//	     	   with an error.
+	//	     	   with an error. This is the DEFAULT mode used 
+	//             by the operator if left empty.
```
Comment (akondur) [inline api/v4/common_types.go:391]: We need to run make generate to generate crd yaml changes here?

File: pkg/splunk/enterprise/afwscheduler_test.go [hunk 2]
```diff
@@ -3048,6 +3048,10 @@ func TestPremiumAppScopedPlaybook(t *testing.T) {
 		t.Errorf("runPlayBook should have returned error")
 	}
 
+	if ! strings.Contains(err.Error(),"premium scoped app package install failed") {
```
Comment (akondur) [inline pkg/splunk/enterprise/afwscheduler_test.go:3051]: Nit: space between '!' and the conditional statement

File: api/v4/common_types.go (hunk 1 above)
Comment (kumarajeet) [reply api/v4/common_types.go:391]: Arjun, I plan to do this after we have `develop` changes in es branch so I have full /complete generated crd yaml file.

File: pkg/splunk/enterprise/afwscheduler_test.go (hunk 2 above)
Comment (kumarajeet) [reply pkg/splunk/enterprise/afwscheduler_test.go]: done

**Pull request merged**